packages/devextreme/js/__internal/ui/calendar/m_calendar.ts

```diff
@@ -1292,7 +1383,12 @@ class Calendar<
     }
 
     const rtlCorrection = this._getRtlCorrection();
-    const offsetSign = offset > 0 ? 1 : offset < 0 ? -1 : 0;
+    let offsetSign = 0;
+    if (offset > 0) {
+      offsetSign = 1;
+    } else if (offset < 0) {
+      offsetSign = -1;
+    }
```

Comment:
[copypaste / code splitting]
```suggestion
    const offsetSign = offset / Math.abs(offset) || 0;
```
or better - use `sign` method from `core/math`

Reply:
https://github.com/DevExpress/DevExtreme/pull/30599/commits/f8f2bf24138a6ed27b92dceaf4ad4b556f536c90